CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 ### Changes since the last release
 
 * Added "Set as Target" in Monitor. (HaullyGames)
+* Connection: connection rate is minimum rate in ControlPath. (HaullyGames)
+* Connection: CommNet consumption was reduced in 84% by Kerbalism, it was too high when compared to RT. (HaullyGames)
```

Comment:
I thought it had been noted that the EC consumption was a problem with RT and not to nerf CommNet